docs/developers/build-vagrant.md

```diff
@@ -0,0 +1,166 @@
+marvel-nccr.fleur : Make fleur executables ------------------------------------------------------------------ 1002.09s
+marvel-nccr.ubuntu_desktop : Install ubuntu-desktop (apt) ---------------------------------------------------- 978.57s
+marvel-nccr.wannier90 : run Wannier90 default tests ---------------------------------------------------------- 972.97s
+marvel-nccr.aiidalab : Install aiidalab ---------------------------------------------------------------------- 749.07s
+marvel-nccr.siesta : Make siesta executables ----------------------------------------------------------------- 422.45s
+marvel-nccr.cp2k : download cp2k binary ---------------------------------------------------------------------- 349.02s
+marvel-nccr.slurm : Install apt packages --------------------------------------------------------------------- 246.30s
+marvel-nccr.simulationbase : Install plotting tools, etc. ---------------------------------------------------- 227.87s
+marvel-nccr.wannier90 : Get Wannier90 source ----------------------------------------------------------------- 203.48s
+marvel-nccr.simulationbase : Install packages for build environment (apt) ------------------------------------ 187.30s
+marvel-nccr.wannier90 : Make Wannier90 executables ----------------------------------------------------------- 179.31s
+marvel-nccr.siesta : Compile "tbtrans" ----------------------------------------------------------------------- 173.52s
+marvel-nccr.simulationbase : Install apt, pip3 --------------------------------------------------------------- 151.31s
+marvel-nccr.aiida : Install DB & more ------------------------------------------------------------------------ 146.25s
+marvel-nccr.wannier_tools : Get wannier_tools source --------------------------------------------------------- 131.27s
+marvel-nccr.ubuntu_desktop : install chromium-browser -------------------------------------------------------- 129.00s
+marvel-nccr.editors : Install some common editors ------------------------------------------------------------ 126.35s
+```
+````
+
+:::{seealso}
+See the continuous deployment (CD) workflow, for up-to-date timings <https://github.com/marvel-nccr/quantum-mobile/actions?query=workflow%3ACD>
+:::
+
+## Prerequisites
+
+- [Vagrant](https://www.vagrantup.com/downloads.html) >= 2.0.1
+- [VirtualBox](https://www.virtualbox.org/wiki/Downloads) >= 6.1.6
+- [Python](https://www.python.org/) >= 3.6
+- Host OS: Building Quantum Mobile has been tested on MacOS, Ubuntu and Windows.
+
+````{dropdown} Building on Windows
+- Install [Windows Subsystem for Linux](https://en.wikipedia.org/wiki/Windows_Subsystem_for_Linux)
+- Install VirtualBox under *Windows*
+- Add the following lines to your `.bashrc` in WSL:
+
+  ```bash
+  # Enable WSL feature in vagrant
+  export VAGRANT_WSL_ENABLE_WINDOWS_ACCESS="1"  
+  # Add virtualbox under Windows to $PATH
+  export PATH="${PATH}:/mnt/c/Program Files/Oracle/VirtualBox"
+  ```
+````
+
+## Customising the build
```

Comment:
Note to self: I'll make a small PR once this is merged, to harmonize this with the "Create the virtual machine" and "customizing QM" sections